src/definitions/behaviors/form.js

```diff
@@ -552,7 +552,7 @@
                             $field        = module.get.field(field.identifier),
                             value         = $field.val(),
                             prompt        = isFunction(rule.prompt)
-                                ? rule.prompt(value)
+                                ? rule.prompt.call($field[0], value) 
```

Comment:
Please remove the trailing space to satisfy the linter ;)
```suggestion
                                ? rule.prompt.call($field[0], value)
```

Reply:
done